sbioapputils/app_runner/app_runner_utils.py

```diff
@@ -116,25 +116,33 @@ def get_job_folder(cls, job_id: str):
  
     @classmethod
     def get_job_config(cls, job_id: str):
-        token = cls.get_api_token()
-        api_url = os.environ.get("SBIO_API_URL")
-        headers = {'Authorization': f'Bearer {token}'}
-        response = requests.get(f'{api_url}/api/jobs/{job_id}/config', headers=headers)
-        if response.status_code == 200:
-            return response.json()['config']          
+        if "JOB_CONFIG" in os.environ:
```

Comment:
Generally, it is better to get all the input variables in a separate file so it is easy to see which input can be provided to the code; like it was done in the backend repository https://github.com/Superbio-ai/superbio_central/blob/main/services/config.py
Ideally, it should be a singleton class so that code is not run every time it is accessed.
I'm not requesting it as a change now but just mentioning it for the future